LWA/LWA_bifrost.py

```diff
@@ -1,65 +1,70 @@
 #!/usr/bin/env python
 
-## Core Python Includes
 from __future__ import print_function
-import signal
-import logging
-import time
-import json
+
+# Set Backend
+import matplotlib
+matplotlib.use("Agg") # noqa
+
+# Core Python Includes
 import os
 import sys
+import json
+import time
+import numpy
+import signal
+import logging
 import threading
 import argparse
-import numpy
-import time
 from collections import deque
 from scipy.fftpack import fft
-import matplotlib
-matplotlib.use('Agg')
-import matplotlib.pyplot as plt
+
 import datetime
 import ctypes
 
-## Profiling Includes
+# Profiling Includes
 import cProfile
 import pstats
 
-## Bifrost Includes
+# Bifrost Includes
+import bifrost
+import bifrost.affinity
 from bifrost.address import Address as BF_Address
 from bifrost.udp_socket import UDPSocket as BF_UDPSocket
 from bifrost.udp_capture import UDPCapture as BF_UDPCapture
 from bifrost.ring import Ring
-from bifrost.unpack import unpack as Unpack
 from bifrost.quantize import quantize as Quantize
-from bifrost.reduce import reduce as Reduce
 from bifrost.proclog import ProcLog
 from bifrost.libbifrost import bf
 from bifrost.fft import Fft
 from bifrost.romein import Romein
-import bifrost
-import bifrost.affinity
-from bifrost.libbifrost import bf
+
```

Comment:
from bifrost.aCorr import aCorr
from bifrost.xCorr import xCorr

Reply:
These modules are tested and part of epic_optim branch which can be used and incorporated.